src/razor/src/diagnostics/reportIssueCreator.ts

```diff
@@ -45,27 +46,55 @@ export class ReportIssueCreator {
         const sanitizedCSharpContent = this.sanitize(csharpContent);
         const sanitizedHtmlContent = this.sanitize(htmlContent);
         const sanitizedDotnetInfo = this.sanitize(dotnetInfo);
-        return `## Is this a Bug or Feature request?:
-Bug
 
-## Steps to reproduce
-------------------- Please fill in this section -------------------------
+        const issueType = l10n.t('Is this a Bug or Feature request?');
+
+        const defaultIssueType = l10n.t('Is this a Bug or Feature request?');
+
+        const reproStepsHeader = l10n.t('Steps to reproduce');
+        const genericPlaceholderContent = l10n.t('Please fill in this section');
+        const problemDescriptionHeader = l10n.t('Description of the problem');
+        const expectedBehaviorHeader = l10n.t('Expected behavior');
+        const actualBehaviorHeader = l10n.t('Actual behavior');
+        const logsHeader = l10n.t('Logs');
+
+        const omniSharpHeader = l10n.t('OmniSharp');
+        const omniSharpInstructions = l10n.t(
+            "To find the OmniSharp log, open VS Code's" + ' "Output" pane, then in the dropdown choose "OmniSharp Log".'
+        );
+        const razorHeader = l10n.t('Razor');
+        const expandText = l10n.t('Expand');
+
+        const workspaceInfo = l10n.t('Workspace information');
+        const razorDocument = l10n.t('Razor document');
+        const projectedCSDocument = l10n.t('Projected CSharp document');
+        const projectedHtmlDocument = l10n.t('Projected Html document');
+        const machineInfo = l10n.t('Machine information');
+        const razorVersion = l10n.t('Razor.VSCode version');
+        const vscodeVersion = l10n.t('VSCode version');
+        const extensionsLabel = l10n.t('Extensions');
```

Comment:
Do you actually want to localize these or leave them in English since we are the ones who have to consume the resulting issue?

Reply:
The Razor team had a discussion on this, and we want it to be easy for the customer to fill in the details, and from the types of content we will see and the relative locations, we should be able to interpret the contents.